frontend/src/firebase/firebaseInteractor.ts

```diff
@@ -40,22 +40,25 @@ export default class FirebaseInteractor {
    * @param {String} uri the uri to the image
    * @returns {Promise<String>}
    */
-  async downloadImage(uri) {
+  async downloadImage(uri : string) : Promise<string> {
     return await this.storage.ref().child(uri).getDownloadURL();
   }
 
   /**
    * Cretes an account for a user
    */
-  async createAccount(email, password) {
+  async createAccount(email : string, password : string) {
     let userAuth = await this.auth.createUserWithEmailAndPassword(
       email,
       password
     );
+
+    // typescript is angry bc userAuth.user could be null
+    // @ts-ignore
     userAuth.user.sendEmailVerification();
```

Comment:
Feel free to change this to:
```suggestion
    userAuth.user?.sendEmailVerification();
```
I think that would fix the issue.